metals/src/main/scala/scala/meta/internal/metals/MetalsEnrichments.scala

```diff
@@ -978,11 +978,37 @@ object MetalsEnrichments
         else diag.getSeverity.toLsp,
         if (diag.getSource == null) "scalac" else diag.getSource,
       )
-      Option(diag.getCode()).foreach { code =>
-        ld.setCode(diag.getCode())
```

Comment:
Just to double check, we no longer call `ld.setCode` in the new version anywhere? :slightly_smiling_face:

Reply:
Good catch, fixed! I was trying to do too many things at the same time 😅

Reply:
No problem, glad to see support for passing through the `Deprecated` diagnostic tag. :+1: